### PR TITLE
(RE-5464) Add Fedora 23 packer configs

### PR DESCRIPTION
**Pull request by heathseals**
This PR adds base and vsphere-nocm packer templates. The vSphere artifact is meant to be used with the vmpooler.

### CODE REVIEW

File: templates/fedora-23/files/ks.cfg [hunk 1]
```diff
@@ -0,0 +1,35 @@
+install
+cdrom
+lang en_US.UTF-8
+keyboard us
+network --bootproto=dhcp
+rootpw puppet
+firewall --disabled
+authconfig --enableshadow --passalgo=sha512
+selinux --disabled
```
Comment (stahnma) [inline templates/fedora-23/files/ks.cfg]: Do we have selinux off on all of our templates?

Comment (shrug) [inline templates/fedora-23/files/ks.cfg]: doh, I just merged. 

I know it's on on the el-7 templates, at least

Comment (heathseals) [inline templates/fedora-23/files/ks.cfg]: @stahnma i doubt it's consistent among the majority of our current templates, but it's disabled on any template generated by packer.

**Pull request merged**